README.md

```diff
@@ -130,7 +163,7 @@ Testing FreeDV API:
   $ play -t .s16 -r 16000 -b 16 out.raw
 ```
 
-## Building and installing on OSX
+## Building and installing on macOS
 
 Please see [README.osx](README.osx).
 
```

Comment:
@drowe67, do we still support FreeBSD at all? Should I spin up a VM with it here and do some tests?

Reply:
Gosh I'm not sure.  Sure, if you like, give it a try.

Reply:
Seems to work for decoding .wav files in single sound card mode:

<img width="1552" alt="Screen Shot 2022-01-02 at 10 04 18 PM" src="https://user-images.githubusercontent.com/449242/147903363-2c5ebc7c-67a8-41e5-8e0f-2f0344bf052d.png">

I didn't try to set up hamlib or a second sound card to test TX, though. Plus LPCNet spit out some errors about /proc/cpuinfo not existing during cmake and incorrectly assumed there was no AVX support.

Reply:
(BTW, build_linux.sh worked on FreeBSD as well once I installed the needed packages using pkg and from Ports.)

Reply:
That's quite impressive!  I guess it shows what we have is quite portable ...

Reply:
I guess the only finishing touch is a BSD version of detecting AVX.